src/nusoap.php

```diff
@@ -3871,11 +3875,11 @@ function service($data)
             } else {
                 $this->debug("In service, there is no WSDL");
                 header("Content-Type: text/html; charset=ISO-8859-1\r\n");
-                print "This service does not provide WSDL";
+                print $this->sanitize("This service does not provide WSDL");
             }
         } elseif ($this->wsdl) {
             $this->debug("In service, return Web description");
-            print $this->wsdl->webDescription();
+            print $this->sanitize($this->wsdl->webDescription());
```

Comment:
This is a huge breaking change: HTML documentation can be embedded in WSDL files, and this completely breaks its display.